config.full.toml

```diff
@@ -0,0 +1,122 @@
+# # Defaults to 300
+# # Env override: ATHENS_TIMEOUT
+# TimeoutSeconds = 300
+
+# # EnableCSRFProtection determines whether to enable CSRF protection.
+# # Defaults to false
+# # Env override: ATHENS_ENABLE_CSRF_PROTECTION
+# EnableCSRFProtection = false
+
+
+# [Proxy]
+#     # Possible values are memory, disk, mongo, postgres, sqlite, cockroach, mysql
+#     # Defaults to mongo
+#     # Env override: ATHENS_STORAGE_TYPE
+#     StorageType = ""
+#     # Env override: PORT
+# 	Port = ""
+#     # Env override: OLYMPUS_GLOBAL_ENDPOINT
+#     OlympusGlobalEndpoint = ""
+#     # Redis queue for buffalo workers
+#     # Defaults to ":6379"
+#     # Env override: ATHENS_REDIS_QUEUE_PORT
+# 	RedisQueueAddress     = ""
+
+# [Olympus]
+#     # Possible values are memory, disk, mongo, postgres, sqlite, cockroach, mysql
+#     # Defaults to memory
+#     # Env override: ATHENS_STORAGE_TYPE
+#     StorageType = ""
+#     # Env override: PORT
+# 	Port = ""
+#     # Background worker type. Defaults to redis
+#     # Env override: OLYMPUS_BACKGROUND_WORKER_TYPE
+#     WorkerType = ""
+#     # Redis queue for buffalo workers
+#     # Defaults to ":6379"
+#     # Env override: OLYMPUS_REDIS_QUEUE_PORT
+#     RedisQueueAddress = ""
+
+# [Storage]
+#     [Storage.CDN]
+#         # Env override: CDN_ENDPOINT
+#         Endpoint = ""
+#         # Defaults to Global Timeout
+#         TimeoutSeconds = 30
+#     [Storage.Disk]
+#         # RootPath is the Athens Disk Root folder 
+#         # Env override: ATHENS_DISK_STORAGE_ROOT
+#         RootPath = ""
+#         # Defaults to Global Timeout
+#         TimeoutSeconds = 30
+#     [Storage.GCP]
+#         # Env overide: GOOGLE_CLOUD_PROJECT
+#         ProjectID = ""
+#         # Env override: ATHENS_STORAGE_GCP_BUCKET
+# 	    Bucket = ""
+#         # Defaults to Global Timeout
+#         TimeoutSeconds = 30
+#     [Storage.Minio]
+#         # Env override: ATHENS_MINIO_ENDPOINT
+#         Endpoint = ""
+#         # Env override: ATHENS_MINIO_ACCESS_KEY_ID
+#         Key = ""
+#         # Env override: ATHENS_MINIO_SECRET_ACCESS_KEY
+#         Secret = ""
+#         # Defaults to Global Timeout
+#         TimeoutSeconds = 30
+#         # Defaults to true
+#         # Env override: ATHENS_MINIO_USE_SSL
+#         EnableSSL = true
+#         # Defaults to gomods
+#         # Env override: ATHENS_MINIO_BUCKET_NAME
+#         Bucket = "gomods"
+#     [Storage.Mongo]
+#         # Env override: ATHENS_MONGO_STORAGE_URL
+#         URL = ""
+#         # Env override: MONGO_USER
+#         User = ""
+#         # Env override: MONGO_PASSWORD
+#         Password = ""
+#         # Defaults to Global Timeout
+#         # Env override: MONGO_CONN_TIMEOUT_SEC
+#         TimeoutSeconds = 30
+#     [Storage.RDBMS]
+#         # Env override: ATHENS_RDBMS_STORAGE_NAME
+#         Name = ""
+#         # Defaults to Global Timeout
+#         TimeoutSeconds = 30
```

Comment:
@rchakra3 is this an example file (if so, that's awesome by the way!)? if so, can you put a comment indicating that at the top?